tests/test_duckdb.py

```diff
@@ -18,40 +18,36 @@
     os.environ,
     clear=True,
 )
-def test_duckdb_core_build_and_export():
+def test_duckdb_core_build_and_export(tmp_path):
```

Comment:
I changed this one from TemporaryDirectory to tmp_path for file system mocking mostly because it makes it slightly easier for me to scrape out the underlying filesystem elements (pytest keeps around the last three of the folders created by the tmp_path fixture), which is useful in this specific case for getting the updated export results without having to chain together a local build system for them outside of unit testing.